octopoes/octopoes/api/bulk_router.py

```diff
@@ -49,3 +51,35 @@ def list_reports(
             reports[recipe_id] = report
 
     return reports
+
+
+@router.get("/object-clients")
+def list_object_clients(
+    clients: set[str] = Query(set()),
```

Comment:
Sets are mutable, so you'd need to use a default factory for this
```suggestion
    clients: set[str] = Query(default_factory=set),
```